### PR TITLE
chore: Move Community Health Files

### PR DESCRIPTION
**Pull request by KieranRobson**
📝Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]`

ℹ️Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

- [x] This PR contains housekeeping only (URL edits, copy changes etc) - if check, delete other lines that don't apply.

This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/main/CONTRIBUTING.md).

This Pull Request should adhere to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/main/CODE_OF_CONDUCT.md).

----

Reference: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file

This PR moves all community health files into the .github folder and updates paths that lead to it. This cleans up the repo while keeping hyperlinks intact

### CODE REVIEW

**Review by dougaitken — APPROVED** [review]

**Comment by dougaitken** [comment]
Hey there, thanks for this, I’d meant to do it myself at some point, it is appreciated.

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 A list of semi to fully remote-friendly companies in or around tech.
 
 Contributions are very welcome! Please
-[submit a pull request on GitHub](https://github.com/remoteintech/remote-jobs/blob/main/CONTRIBUTING.md).
+[submit a pull request on GitHub](https://github.com/remoteintech/remote-jobs/tree/main/.github).
```
Comment (dougaitken) [inline README.md:6]: This is no longer linking the the contributing file, can you fix?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Please send a Pull Request if you have any additions or changes.
 
 Pull requests will be accepted based on the following:
 
-1. The pull request adheres to the repository's [Code of Conduct](/CODE_OF_CONDUCT.md).
+1. The pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/tree/main/.github).
```
Comment (dougaitken) [inline .github/CONTRIBUTING.md:7]: File name is missing from the end of the path

**Review by dougaitken — CHANGES_REQUESTED** [review]

**Comment by dougaitken** [comment]
Two tweaks needed, thanks @KieranRobson

**Comment by KieranRobson** [comment]
Changes made!

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ Please send a Pull Request if you have any additions or changes.
 
 Pull requests will be accepted based on the following:
 
-1. The pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/tree/main/.github).
+1. The pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/tree/main/.github/CONTRIBUTING.md).
```
Comment (dougaitken) [inline .github/CONTRIBUTING.md:7]: Sorry @KieranRobson - this should be linking to the CoC not contributing. I’d change it myself but I can’t edit the file.

**Review by dougaitken — CHANGES_REQUESTED** [review]

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (KieranRobson) [reply .github/CONTRIBUTING.md]: Its been a rough morning 😆, this has now been fixed!

**Review by dougaitken — APPROVED** [review]

**Comment by KieranRobson** [comment]
I see I have been assigned, is there anything else required? I had a quick skim through the changes and cant see anything that I missed.

I wish I did a branch for this so I could work on the next PR 😆

**Pull request merged**